frontend/lib/update-user-privilege.ts

```diff
@@ -8,7 +8,7 @@ export const updateUserPrivilege = async (
   const body = { isAdmin };
 
   const response = await fetch(
-    `${userServiceUri(window.location.hostname)}/users/${id}/privilege`,
+    `${userServiceUri(window.location.hostname, AuthType.Public)}/users/${id}/privilege`,
```

Comment:
Not sure if I understand correctly, but should this be `AuthType.Public`? Updating user privilege should be an admin function?